.eslintignore

```diff
@@ -2,7 +2,6 @@
 **/expected.js
 _output
 test/*/samples/*/output.js
-node_modules
```

Comment:
i wonder why we are not ignoring `node_modules` now?

Reply:
It's the default behaviour of eslint :)